### ISSUE TITLE
LD4 2023 Conference Proposals due June 2 - Do we have ideas?

### ISSUE THREAD

**Issue by lake44me** [uwlib-cams/MARC2RDA#393]
@CECSpecialistI @gerontakos @AdamSchiff @briesenberg07 @GordonDunsire @lake44me @SitaKB @pan-zhuo @szapoun @JianPLee @junghaelee @cwarc

Who has ideas for a proposal?  Who would be willing to present?

**Comment by lake44me** [comment]
Idea #1 - presentation based on selected issues we have wrestled with (or are still wrestling with !) in our mapping.

**Comment by CECSpecialistI** [comment]
> Idea #1 - presentation based on selected issues we have wrestled with (or are still wrestling with !) in our mapping.

I like this idea and would be willing to help present on it.

**Comment by gerontakos** [comment]
I agree. I would like to do something here also. However, the presentations for LD4 are traditionally 20-30 minutes. Perhaps we should plan a few talks on our issues/conversations/decisions (rather than have one where three people talk for only 10 minutes each). I believe we have plenty of content that would generate plenty of interest. It could be designed as a workshop, but I'm worried that would require tremendous effort. Workshop attendees have high expectations! Maybe we should start putting a "workshop" together at the same time as we plan these presentations so that when the next major CFP comes along, we can offer an RDA (or RDA/MARC or ontology matching or whatever) workshop of some kind. Is that LD4 conference plan too ambitious? (I know the workshop idea is a little overambitious.) It will take a little discussion to figure out who will do what.

**Comment by lake44me** [comment]
I have several ideas but haven't had time to flesh out...  maybe this weekend.  One that popped into my head - prerecord presentation (15-20 mins) with multiple presenters (everyone who wants to, pick favorite issues and do 3-4 mins.). Maybe one person gives context (we're mapping for U. of Washington, but with an eye to sharing up to larger communities; we argue and vote if needed, we make provisional decisions to revisit later; biggest tension - trying to map all data - vs. uncertainty/ value judgments or whatever.   Then, big Q/A for 10 mins.  

in other words, high level flyover but try to get as deep as you can to the core of an issue in 3 mins. 
 
Or, we could pick like 3 issues to delve in, but mention the others.

**Comment by lake44me** [comment]
I woke up this morning thinking about our discussion/decision on Enhanced 505.  I think it would make a good short presentation.    Somewhere there would be a slide "Aggregates!  Gggrrr!  (Choke)".  And a lesson "MARC shortcuts short-circuit in RDA linked data?"

**Comment by CECSpecialistI** [comment]
The 505 would be an interesting one, so would reproductions. What about...admin and provenance metadata? That one might be fun because it would give us a chance to spend more time on something we haven't "moved on" from. So would aggregates in general. Or...identifiers. This conference is coming up...let's devote a few minutes to it during the meeting next week and see if we can come up with something.

**Comment by CECSpecialistI** [comment]
I also like the idea of a few talks. We could each pick our favorite messy problem.

**Comment by lake44me** [comment]
Here's my proposal for 1/2 hour:

UW MARC to RDA mapping challenges: 505 + $5

"When we were climbing up the stair, we met a data that wasn't there..."  The University of Washington MARC2RDA mapping project aims to get us past the huge sticking point for library linked data - transforming legacy MARC so it will be usable in the future.  The group has found many mapping decisions challenging because it's hard to connect the dots when parts of data needed to support RDA in RDF just aren't there in MARC21.  How far can or should we go to capture data that supports discovery or other functionality in MARC?  Should we create new entities, extrapolate values from our understanding of rules and experience, "just create a note field", or ignore some data?  Here's a look inside the decision process for two MARC fields - "enhanced" tables of contents (505 _0 to catalogers) and subfield 5 (institution to which the field applies).  These are preliminary decisions - how would you call it or code it?

**Comment by CECSpecialistI** [comment]
I'm running low on time, and don't have any ideas I'm dying to talk about at LD4, so I might sit this one out.

**Comment by gerontakos** [comment]
I'll submit a proposal to discuss IRIs and the confusion for GLAM implementers of LD. The most concrete part of the talk will be IRIs in the context of RDA and the difficulties we encountered during the mapping, but the focus will be on IRIs, not our project, while, at the same time, our project will be the featured feature.

**Comment by CECSpecialistI** [comment]
I like it. Thank you Theo!
